splinepy/io/irit.py

```diff
@@ -0,0 +1,35 @@
+"""
+irit io.
+"""
+from splinepy import splinepy_core
+
+
+def load(fname):
+    """
+    Read spline in `.itd` form.
+
+    Parameters
+    -----------
+    fname: str
+
+    Returns
+    --------
+    splines: list
+    """
+    return splinepy_core.read_irit(fname)
```

Comment:
Write issue, please, to replace tabs by spaces

Reply:
#69